cubeviz/data_factories/__init__.py

```diff
@@ -58,6 +58,19 @@ def name(self):
     def type(self):
         return self._type
 
+    def get_units(self, header):
+        """
+        Extract BUNIT from header
+        :param header: header
+        :return: str: Shortened unit
+        """
+        units = str(header['BUNIT'])
+
+        # Unit label shorten depending on data type here
+        if 'MANGA' == self.type:
+            units = units.replace("Ang", "A")
```

Comment:
It's maybe not that important right now, but it would probably be useful to encode this replacement logic in the data factory configuration file somehow.

Reply:
Done.